src/core/Address.ts

```diff
@@ -74,9 +74,9 @@ export abstract class Address {
             }
             try {
                 const address = SymbolAddress.createFromString(encodedAddress);
-                const hasher = new SHA3(256);
-                const hash = hasher.update(Buffer.from(address.rawAddress.addressWithoutChecksum)).digest();
-                return Converter.uint8ToHex(hash.subarray(0, 3)) === Converter.uint8ToHex(address.rawAddress.checksum);
+                const hasher = sha3_256.create();
+                const hash = hasher.update(address.rawAddress.addressWithoutChecksum).arrayBuffer();
+                return new Uint8Array(hash).subarray(0, 3).toString() === address.rawAddress.checksum.toString();
```

Comment:
why toStrings ?

Reply:
otherwise, have to deep compare arrays?

Reply:
isn't comparing arrays byte-by-byte less work than converting them to strings and then comparing the strings byte-by-byte?